blocksuite/affine/shared/src/services/index.ts

```diff
@@ -19,3 +19,4 @@ export * from './sidebar-service';
 export * from './telemetry-service';
 export * from './theme-service';
 export * from './virtual-keyboard-service';
+export * from './toolbar-service';
```

Comment:
I think we should put toolbar service into toolbar widget package. And let other blocks dependent on that package.